pybamm/parameters/lithium_ion_parameters.py

```diff
@@ -142,45 +134,56 @@ def _set_dimensional_parameters(self):
         self.Q_Li_init = self.n_Li_init * self.F / 3600
 
         # Reference OCP based on initial concentration
-        self.ocv_ref = self.p.U_ref - self.n.U_ref
-        self.ocv_init_dim = self.p.prim.U_init_dim - self.n.prim.U_init_dim
+        self.ocv_init = self.p.prim.U_init - self.n.prim.U_init
+
+        # Some scales
+        self.thermal_voltage = self.R * self.T_ref / self.F
+        self.I_typ = self.Q / (self.A_cc * self.n_electrodes_parallel)
+        self.a_j_scale = self.I_typ / self.L_x
 
-    def chi_dimensional(self, c_e, T):
+    def chi(self, c_e, T):
         """
         Thermodynamic factor:
             (1-2*t_plus) is for Nernst-Planck,
             2*(1-t_plus) for Stefan-Maxwell,
         see Bizeray et al (2016) "Resolving a discrepancy ...".
         """
-        return (2 * (1 - self.t_plus_dimensional(c_e, T))) * (
-            self.one_plus_dlnf_dlnc_dimensional(c_e, T)
-        )
+        return (2 * (1 - self.t_plus(c_e, T))) * (self.one_plus_dlnf_dlnc(c_e, T))
 
-    def t_plus_dimensional(self, c_e, T):
-        """Cation transference number (dimensionless)"""
+    def chiRT_over_Fc(self, c_e, T):
+        """
+        chi * (1 + Theta * T) / c,
+        as it appears in the electrolyte potential equation
+        """
+        tol = pybamm.settings.tolerances["chi__c_e"]
+        c_e = pybamm.maximum(c_e, tol)
+        return (self.R * T / self.F) * self.chi(c_e, T) / c_e
+
+    def t_plus(self, c_e, T):
+        """Cation transference number"""
         inputs = {"Electrolyte concentration [mol.m-3]": c_e, "Temperature [K]": T}
         return pybamm.FunctionParameter("Cation transference number", inputs)
 
-    def one_plus_dlnf_dlnc_dimensional(self, c_e, T):
-        """Thermodynamic factor (dimensionless)"""
+    def one_plus_dlnf_dlnc(self, c_e, T):
```

Comment:
Maybe for a separate (small) issue, should we rename this to `thermodynamic_factor`? It is not much longer, and more informative.

Reply:
yes :) I didn't know what it was called at the time haha